### PR TITLE
Added docs for Postgres DB configuration

### PR DESCRIPTION
**Pull request by justo-mend**

### CODE REVIEW

File: docs/configure-postgres-db.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+To configure renovate to use a PostgreSQL database, the following environment variables should be supplied to the Server containers (not required for Worker environment config).
+
+| Environment variable name | Description |
+|---|---|
+| MEND_RNV_DATA_HANDLER_TYPE | Set to ‘postgresql’ to use a PostgreSQL database |
+| PGDATABASE | Name of the database instance. Eg. ‘postgres’ |
+| PGUSER | Postgres User name. Must have Create Schema permission. |
+| PGPASSWORD | Postgres User password |
+| PGHOST | Host name of the PostgreSQL instance |
+| PGPORT | Host Port for the PostgreSQL instance |
```
Comment (rarkins) [inline docs/configure-postgres-db.md:12]: Do any of the above have default values so are optional?

File: docs/configure-postgres-db.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Example configuration in a Docker Compose file is shown below.
+Swap out the values of PGXXX parameters for your own instances of the PostgreSQL database.
+
+```
+…
+  rnv-ee-server:
+    restart: always
+    image: ghcr.io/mend/renovate-ee-server
+    deploy:
+      replicas: 2
```
Comment (rarkins) [inline docs/configure-postgres-db.md:27]: Leave HA separate

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/configure-postgres-db.md (hunk 2 above)
Comment (justo-mend) [reply docs/configure-postgres-db.md:27]: Will remove "deploy" config.

File: docs/configure-postgres-db.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Postgres DB Configuration
```
[the remark this answers is not in the document]
Comment (justo-mend) [reply docs/configure-postgres-db.md:1]: Will put that on the TODO list.

**Review by nabeelsaabna — DISMISSED** [review]

File: docs/configure-postgres-db.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Details of the PostgreSQL user, password, host and port must match your hosted instance of the Renovate PostgreSQL DB.
+This could be an externally managed PostgreSQL server, or an instance started inside your Docker Compose configuration.
+
+## Spinning up a PostgreSQL DB container
+
+For convenience, you can spin up your own instance of a PostgreSQL DB, and an  optional frontend Web UI.
+
+###  Docker Compose example
+An example of starting a PostgreSQL DB server container in a Docker Compose file is shown below.
```
Comment (rarkins) [inline docs/configure-postgres-db.md:48]:
```suggestion

An example of using a PostgreSQL DB server container with Docker Compose is shown below.
```

File: docs/configure-postgres-db.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+To configure renovate to use a PostgreSQL database, the following environment variables should be supplied to the Server containers (not required for Worker environment config).
+
+| Environment variable name | Description |
+|---|---|
+| MEND_RNV_DATA_HANDLER_TYPE | Set to ‘postgresql’ to use a PostgreSQL database |
+| PGDATABASE | Name of the database instance. Eg. ‘postgres’ |
+| PGUSER | Postgres User name. Must have Create Schema permission. |
+| PGPASSWORD | Postgres User password |
+| PGHOST | Host name of the PostgreSQL instance |
+| PGPORT | Host Port for the PostgreSQL instance |
```
Comment (rarkins) [inline docs/configure-postgres-db.md:12]:
```suggestion
| Environment variable name  | Description                                             |
|----------------------------|---------------------------------------------------------|
| MEND_RNV_DATA_HANDLER_TYPE | Set to ‘postgresql’ to use a PostgreSQL database        |
| PGDATABASE                 | Name of the database instance. Eg. ‘postgres’           |
| PGUSER                     | Postgres User name. Must have Create Schema permission. |
| PGPASSWORD                 | Postgres User password                                  |
| PGHOST                     | Host name of the PostgreSQL instance                    |
| PGPORT                     | Host Port for the PostgreSQL instance                   |
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/configure-postgres-db.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+This could be an externally managed PostgreSQL server, or an instance started inside your Docker Compose configuration.
+
+## Spinning up a PostgreSQL DB container
+
+For convenience, you can spin up your own instance of a PostgreSQL DB, and an  optional frontend Web UI.
+
+###  Docker Compose example
+
+An example of using a PostgreSQL DB server container with Docker Compose is shown below.
+```
```
Comment (rarkins) [inline docs/configure-postgres-db.md:50]:
```suggestion

```
```

**Review by rarkins — APPROVED** [review]

**Review by nabeelsaabna — APPROVED** [review]

**Pull request merged**